workers/generic-worker/schemas/multiuser_posix.yml

```diff
@@ -269,6 +269,10 @@ properties:
           title: Exit codes
           type: integer
           minimum: 1
+  log:
+    title: Livelog artifact name
```

Comment:
I've just realised, the description is probably needed after all, to explain that this setting affects both the name of the live log, _and_ the name of the backing log. It should explain how the backing log name is derived from the live log artifact name.